admin/README.md

```diff
@@ -160,3 +160,17 @@ after making changes ensure you do `docker-compose build` to rebuild from source
 In any service methods which updates tables which will affect these caches, `redisCacheService.drop('cacheName')` can be used to drop it for re-querying. It accepts a single string or an array of strings.
 
 Updates to an existing Redis cache can be done with `redisCacheService.update`. It will perform the supplied changes object on the Redis cache and then send it (with the affected table name) to the Azure `sql-update` message queue. Where it will then be consumed and applied in SQ server by a listener in `/functions`.
+
+## Error Information
+
+When triggering an error, you can specify a dedicated error type that includes the actual error and a user friendly message as arguments.
+
+These error types will need to be generated and extend from `mtcBaseError`. For example:
+```
+checkWindowV2Service.getCheckWindow = async (urlSlug) => {
+  if (!urlSlug || !validate(urlSlug)) {
+    throw new MtcCheckWindowNotFoundError(mtcCheckWindowNotFoundErrorMessages.errorMessage, mtcCheckWindowNotFoundErrorMessages.userMessage)
```

Comment:
could we replace the mtcCheckWindowNotFoundErrorMessages messages type with an actual string, to make it a more obvious example of what this enables us to do.